robots-module/src/main/java/orwell/tank/INewValueVisitor.java

```diff
@@ -0,0 +1,13 @@
+package orwell.tank;
+
+import lejos.mf.common.UnitMessage;
+
+/**
+ * Created by Michaël Ludmann on 6/10/15.
+ */
+public interface INewValueVisitor extends ITankVisitor {
```

Comment:
The goal of this class is not very clear. Maybe it gets clearer after reading the whole program ...